libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -1250,6 +1251,34 @@ bool RC_Channel::do_aux_function(const aux_func_t ch_option, const AuxSwitchPos
         break;
     }
 
+    case AUX_FUNC::ARM_EMERGENCY_STOP: {
+        switch (ch_flag) {
+           case AuxSwitchPos::HIGH:
+              SRV_Channels::set_emergency_stop(false);
+              AP::arming().arm(AP_Arming::Method::AUXSWITCH, true);
+              if (hal.util->get_soft_armed()) {
```

Comment:
This always disables E stop so it should always log?

Reply:
just following E-Stop..logs on both true and false